server/lib/builders/document-builder.js

```diff
@@ -88,18 +88,31 @@ module.exports = exports = class DocumentBuilder {
 	removeNonSyndicatableImages() {
 
 		const embedsMap = arrayToMap(this.content.embeds);
-
 		Array.from(this.contentDocument.getElementsByTagName('img')).forEach(
 			(el) => {
-				const imageType = el.getAttribute('data-image-type');
-
+				let isFlourishElement = false;
+				// identify flourish element
+				const elementSrc = el.getAttribute('src');
+				if(elementSrc?.includes('public.flourish.studio/')) {
```

Comment:
Comment/suggestion: I'll consider a more functional approach instead of imperative style when making changes in the DOM. We could simplify this to something like:

```
// Determine if it is a Flourish Element
const isFlourishElement = elementSrc?.includes('public.flourish.studio/');
```

```
// Determine image type
const imageType = isFlourishElement
? 'graphic'
: el.getAttribute('data-image-type');
```
This reduces multiple declarations and less to maintain.